src/browser/main.js

```diff
@@ -178,6 +178,17 @@
 
         // Abandonware OS images are from https://winworldpc.com/library/operating-systems
         var oses = [
+            {
+                id: "debian",
+                name: "Debian",
+                memory_size: 512 * 1024 * 1024,
+                vga_memory_size: 8 * 1024 * 1024,
+                state: { url: host + "debian-state-base.bin" },
+                filesystem: {
+                    "baseurl": host + "debian-9p-rootfs-flat/",
+                    "basefs": { url: host + "debian-base-fs.json" }
+                }
+            },
```

Comment:
The problem with this is that it will create a broken debian profile on the deployed version of v86. One of the following would be better:

1. Add a debian example in the examples directory
2. Add this change to the readme

I'd prefer 1. but would accept 2. too.

Reply:
did both, how does that look?

Reply:
Looks good, thanks!